corpus/async.txt

```diff
@@ -0,0 +1,54 @@
+============================================
+Async function
+============================================
+
+async fn abc() {}
+
+async fn main() {
+    let x = futures.await?;
+}
+
+---
+
+(source_file
+  (function_item
+    (function_modifiers)
+    (identifier)
+    (parameters)
+    (block))
+  (function_item (function_modifiers) (identifier) (parameters)
+    (block
+      (let_declaration (identifier) (try_expression
+        (await_expression (identifier)))))))
+
+============================================
+Await expression
+============================================
+
+futures.await;
+
+futures.await?;
+
+futures.await?.await?;
+
+futures.await?.function().await?;
+
+
```

Comment:
Nitpick: could you 🔥 this second blank line? Also, I think you could kill the blank lines between each statement.

Reply:
Sure